ee/clickhouse/models/property.py

```diff
@@ -375,13 +375,14 @@ def get_property_string_expr(
     :return:
     """
 
-    for reserved_words in EVENT_ATTRIBUTE_RESERVED_PROPERTIES_BY_TYPE.values():
-        if property_name in reserved_words:
-            return property_name, False
+    table_string = f"{table_alias}." if table_alias != None else ""
 
-    materialized_columns = get_materialized_columns(table) if allow_denormalized_props else {}
+    if table == "events":
```

Comment:
Is reserved properties just applicable to events, or is it something we'd also want to do for other tables, but we just want to limit scope for now. Which ever way, somehow documenting would be useful

Reply:
Yep, limiting scope for now. But I'd guess we want to do this across tables - the user probably doesn't care if we store something in the properties blob if it's useful for them to filter by

Was going to add documentation to the API docs. Is there a better place?

Reply:
API docs makes sense. I would also put something right here to clarify as if I'm making a change here, this is really were I would like the context. I can just imagine someone coming to the code and thinking why not just do this for all tables, removing the if, luckily the tests will catch this now but will be a big feedback cycle.